conda/recipes/cudf/meta.yaml

```diff
@@ -29,7 +29,7 @@ requirements:
     - rmm {{ minor_version }}.*
   run:
     - python
-    - pandas 1.0.3
+    - pandas >=1.0,<1.1
```

Comment:
We may want to revisit this before 0.15 release to see if we want to support `pandas >=1.0,<2.0` instead. Technically they shouldn't introduce any API breaking changes, but we're also using internal functions at this point.

Reply:
This should be `>1.0,<1.1.0a0` since it looks likes conda-forge does publish RCs to their channel.